src/track/track.cpp

```diff
@@ -137,11 +137,11 @@ void Track::setTrackMetadata(
                 &m_record.refMetadataSynchronized(),
                 !metadataSynchronized.isNull());
         bool modifiedReplayGain = false;
-        if (m_record.getMetadata() != trackMetadata) {
+        if (m_record.getMetadata() != importedMetadata) {
             modifiedReplayGain =
                     (m_record.getMetadata().getTrackInfo().getReplayGain() != newReplayGain);
-            m_record.setMetadata(std::move(trackMetadata));
-            // Don't use trackMetadata after move assignment!!
+            m_record.setMetadata(std::move(importedMetadata));
+            // Don't use importedMetadata after move assignment!!
```

Comment:
Can't we delete this comment?

Reply:
Of course, it is redundant. But since the data of this argument is moved away in the middle of a large scope, this might serve as a reminder to future developers. Unfortunately C++ does not prevent you from doing unintended things as Rust would do.

Reply:
This code has just been moved around, nothing changed.

Reply:
OK